cmd/apply.go

```diff
@@ -47,21 +47,23 @@ const (
 	GcStrategyIgnore = "ignore"
 )
 
+var applyShortDesc = `Apply local Kubernetes manifests (components) to remote clusters`
```

Comment:
Instead of `var`, can you add this as a `const` alongside the others at the top-of-file? And use double quotes instead of ` since there's no characters needing escaping.